### ISSUE TITLE
Stick item on bottom of screen

### ISSUE THREAD

**Issue by parsidev** [emilsjolander/StickyScrollViewItems#39]
Hello.

can you help me to change position of stick on this library?

I need to stick a view on bottom of screen.....

**Comment by Spentas** [comment]
@parsidev do u wanna have something like the normal sticky header but in reverse? how many sticky items u have?

**Comment by parsidev** [comment]
@Spentas yes i want to stick item on bottom of screen. i have one sticky item

**Comment by Spentas** [comment]
for one item, why dont u stick a separate view at the bottom of ur list view? or use a floating action button and trigger it to comes up when u reach to a certain position of your list view

**Comment by Varun7691** [comment]
Hey did you guys come up with a solution for this ?! I am in need of the same thing. Any Help ?!
